front/src/config/i18n/en.json

```diff
@@ -13,7 +13,20 @@
     "percent": "%",
     "degreeValue": "{{value}}°",
     "workInProgress": "Work in progress...",
-    "save": "Save"
+    "save": "Save",
+    "daysOfTheWeek": {
+      "monday": "Monday",
```

Comment:
dayjs can't handle it?

https://day.js.org/docs/en/display/format

Reply:
And this to list days

https://day.js.org/docs/en/i18n/listing-months-weekdays

Reply:
For more, it's the second day list in i18n file...